extension/src/extension.ts

```diff
@@ -193,6 +193,10 @@ class Extension extends Disposable {
     )
     registerPipelineCommands(this.pipelines, this.internalCommands)
     registerPlotsCommands(this.plots, this.internalCommands, this.setup)
+    this.internalCommands.registerExternalCommand(
```

Comment:
[I] Plots commands should be registered in `extension/src/plots/commands/register.ts`.